### DISCUSSION TITLE
Cordova

### DISCUSSION THREAD

**Discussion by jasonz1987** [byteee-fund/bsf-crowdfunding-plan#3] [vote]
If you have other projects that need voting, please leave a comment directly under this discussion。

project demo:

**project name：**

ex： cordova-plugin-xxx

**repo url：**

ex: https://github.com/xxxx

**reason：**

1、 not maintained

2、 have some bugs

Projects that have entered the crowdfunding period

- ~~cordova-plugin-fingerprint-aio~~ #5 
- ~~cordova-plugin-app-version~~ #5 
- ~~cordova-plugin-android-permissions~~ #5

**Comment by jiangzik** [comment]
project name：cordova-plugin-jitsi

repo url：https://github.com/YPT-ME/cordova-plugin-jitsi

reason：

1、 not maintained

2、 Android 12 started, unavailable

**Reply by jasonz1987** [reply]
he project has entered the crowdfunding period, please check for details https://github.com/byteee-fund/bsf-crowdfunding-plan/discussions/5

**Comment by NityaSantosh26** [comment]
project name：cordova-plugin-keyboard

repo url：https://github.com/cjpearson/cordova-plugin-keyboard

reason: Not Maintained
Issues still exists in most of iOS devices

**Reply by jasonz1987** [reply]
The project has entered the crowdfunding period, please check for details [https://github.com/byteee-fund/bsf-crowdfunding-plan/discussions/5](https://github.com/byteee-fund/bsf-crowdfunding-plan/discussions/5)
